ofrak_core/setup.py

```diff
@@ -39,11 +39,13 @@ def run(self):
     install_requires=[
         "beartype~=0.10.2",
         "fdt==0.3.2",
+        "importlib-metadata>=1.4",
         "intervaltree==3.1.0",
         "keystone-engine==0.9.2",
         "lief==0.12.2",
-        "ofrak_io",
-        "ofrak_type",
+        "ofrak_io~=1.0",
+        "ofrak_type~=2.0",
+        "ofrak_patch_maker~=2.0",
```

Comment:
Yes, `ofrak_io~=1.0.0` is semantically different from `ofrak_io~=1.0`, the former would exclude `1.1` while the latter would include it. Since minor version bumps are backwards-compatible, this expresses the actual compatibility of these packages.